pyproject.toml

```diff
@@ -68,12 +68,21 @@ source = "vcs"
 
 [project.optional-dependencies]
 tests = ["hypothesis", "pytest"]
-dev = ["pre-commit", "Jinja2"]
+dev = ["pre-commit", "Jinja2", "setuptools", "playwright"]
 
 [tool.hatch.build.targets]
 sdist = { exclude = ["/.github", "/docs", "/test", "/scripts"] }
 wheel = { packages = ["a11y_pygments"] }
 
+[tool.hatch.envs.dev]
+features = ["dev"]
```

Comment:
Previously I don't think the "dev" optional-dependencies (line 71) above were getting installed in the Hatch dev environment. I believe they have to be linked using this `features` key.